src/OpenLoco/Core/LocoFixedVector.hpp

```diff
@@ -4,31 +4,30 @@
 
 namespace OpenLoco
 {
-    template<typename ValueType>
-    class LocoFixedVector
```

Comment:
With LocoFixedVector effectively being renamed to FixedVector, perhaps we should rename the file as well.